### PR TITLE
Airdrop database migrations

### PR DESCRIPTION
**Pull request by rajeevbbqq**

### CODE REVIEW

File: alembic/versions/700e63da6309_airdroptables.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('airdrop',
+    sa.Column('row_id', sa.INTEGER(), autoincrement=False, nullable=False),
+    sa.Column('address', sa.VARCHAR(length=50), nullable=False),
+    sa.Column('org_name', sa.VARCHAR(length=256), nullable=False),
+    sa.Column('token_name', sa.VARCHAR(length=50), nullable=False),
```
Comment (raamb) [inline alembic/versions/700e63da6309_airdroptables.py:25]: Lets increase token name to 128 characters

File: alembic/versions/700e63da6309_airdroptables.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    sa.Column('row_updated', mysql.TIMESTAMP(), server_default=sa.text('CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP'), nullable=False),
+    sa.Column('airdrop_window_id', sa.INTEGER(), nullable=False),
+    sa.Column('rule', sa.TEXT(), nullable=False),
+    sa.PrimaryKeyConstraint('row_id')
+    )
+    op.create_table('claim_history',
+    sa.Column('row_id', sa.BIGINT(), autoincrement=True, nullable=False),
+    sa.Column('row_created', mysql.TIMESTAMP(), server_default=sa.text('CURRENT_TIMESTAMP'), nullable=False),
+    sa.Column('row_updated', mysql.TIMESTAMP(), server_default=sa.text('CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP'), nullable=False),
+    sa.Column('airdrop_window_id', sa.INTEGER(), nullable=False),
```
Comment (raamb) [inline alembic/versions/700e63da6309_airdroptables.py:64]: Dont we need airdrop_id as well in this table?

File: alembic/versions/700e63da6309_airdroptables.py (hunk 1 above)
Comment (rajeevbbqq) [reply alembic/versions/700e63da6309_airdroptables.py]: Updated in the latest commit @raamb

File: alembic/versions/700e63da6309_airdroptables.py (hunk 2 above)
Comment (rajeevbbqq) [reply alembic/versions/700e63da6309_airdroptables.py]: I was thinking another way by leveraging with SQL queries we get the `airdrop _id` but that will be a heavy operation  @raamb  


Added the `airdrop_id` in the latest commit.

**Review by anandrgitnirman — COMMENTED** [review]
General comments 
- Some of your tables are missing row_id , can we add this consistently across all tables 
- Can you also define the foreign key constraits
- define unique constraints on tables whereever appliacble

**Comment by rajeevbbqq** [comment]
@anandrgitnirman 

- Every table has row_id, row_created, row_updated. Which is extending from AuditClass to each model for avoiding code duplications.
- Added foreign keys in the latest commit
- Added unique constraints in the latest commit

**Review by anandrgitnirman — COMMENTED** [review]
lets also add created_date and updated_date on all the tables (where ever missing) , and make these as last columns ( to give a consistent feel)

**Comment by rajeevbbqq** [comment]
Instead of `created_at, updated_at` we are using `row_created & row_updated` on every columns.

**Review by anandrgitnirman — APPROVED** [review]

**Pull request merged**